### PR TITLE
README fixes, add add file for variables, and separate out profiles m…

### PR DESCRIPTION
**Pull request by chiayi**
…d, and minor fixes.

### CODE REVIEW

File: jupyter-on-gke/iap_module/variables.tf
Comment (yiyinglovecoding) [inline jupyter-on-gke/iap_module/variables.tf:1]: can you also do `terraform fmt`  in this directory? This can auto-format all the files

Comment (yiyinglovecoding) [reply jupyter-on-gke/iap_module/variables.tf:1]: under this file path as well /ai-on-gke/jupyter-on-gke/variables.tf

File: jupyter-on-gke/profiles.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        guarantee: .5
+    memory:
+        limit: 1G
+        guarantee: 1G
+```
+
+The _limit_ if the reousrce sets a hard limit on how much of that resource can the user have.
+The _guarantee_ meaning the least amount of resource that will be available to the user at all times.
+
+Similar to overriding images, the resources can also be overwritten by using `kubesparner_override`:
```
Comment (imreddy13) [inline jupyter-on-gke/profiles.md:73]: typo: kubespawner

File: jupyter-on-gke/profiles.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            command:
+            - "sh"
+            - "-c"
+            - >
+                gitpuller https://github.com/data-8/materials-fa17 master materials-fa;
+```
+
+### Additional Overrides
+
+With `kubespanwer_override` there are additional overrides that could be done, including `lifecycle_hooks`, `storage_capcity`, and `storage class`
```
Comment (imreddy13) [inline jupyter-on-gke/profiles.md:138]: typo: capacity

**Review by imreddy13 — APPROVED** [review]

**Pull request merged**